src/main/java/org/symphonyoss/symphony/messageml/elements/Select.java

```diff
@@ -159,14 +159,22 @@ public void updateBiContext(BiContext context) {
     this.putOneIfPresent(attributesMapBi, BiFields.REQUIRED.getValue(), REQUIRED_ATTR);
     this.putOneIfPresent(attributesMapBi, BiFields.MULTI_SELECT.getValue(), MULTIPLE_ATTR);
 
+    attributesMapBi.put(BiFields.OPTIONS_COUNT.getValue(), countChildrenOfType(Option.class));
```

Comment:
Why not using `putOneIfPresent` as for other cases?

Reply:
the difference is that for the `putOneIfPresent` we are going to check if the element contains a specific value for example `multi_select=true` and if so we are going to add an entry with value set to 1 (acting as a boolean 0/1 if present or not).

In this case instead we want to actually add the counter for how many options we have inside the <select> tag and for the second one we need to check before adding 0 or 1 (it's not that straight forward as for the other ones).